public_suffix_list.dat

```diff
@@ -13690,4 +13690,8 @@ basicserver.io
 virtualserver.io
 enterprisecloud.nu
 
+// Smoove.io : https://www.smoove.io/
```

Comment:
Move this up higher in the PRIVATE DOMAINS section so that it is alphabetically ordered by your org's name please

Reply:
Done 👍